src/cache/backfill.ts

```diff
@@ -43,6 +43,25 @@ export async function cacheHash(
   return null;
 }
 
+export async function cacheFetchWithLocalFallback(
+  hash: string | null,
+  id: string,
+  cwd: string,
+  cacheOptions: CacheOptions
+) {
+  if (!hash) {
+    return false;
+  }
+
+  const remoteCacheOptions =  getCacheConfig(cwd, cacheOptions);
```

Comment:
Shouldn't it try backfilling from local first, and then if miss, try remote?

Reply:
Could we change the configuration instead and allow the user to specify an ordered list of caches? If they want to do local -> remote or remote -> local, should `lage` care?

Reply:
This is a really good point - in addition, should we simply download the remote cache into our local cache?